VacaYAY/BusinessLogicLayer/Exceptions/BadRequestException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLogicLayer.Exceptions
+{
+    public class BadRequestException : Exception
+    {
+        public BadRequestException(string message)
```

Comment:
This may be overkill.

Please check built-in exceptions in the framework: 
https://www.completecsharptutorial.com/basic/complete-system-exception.php